src/client/rspamc.cxx

```diff
@@ -823,6 +825,59 @@ add_options(GQueue *opts)
 	}
 }
 
+static void
+print_indented_line(FILE *out, std::string line, size_t maxlen, size_t indent)
+{
+	if (maxlen < 1) return;
+
+	std::string s;
+	for (size_t pos = 0; pos < line.length(); pos += s.length()) {
+		s = line.substr(pos, pos ? (maxlen-indent) : maxlen);
+		if (indent && pos) fmt::print(out, "{:>{}}", " ", indent);
+		fmt::print(out, "{}\n", s);
+	}
+}
+
+static void
+rspamc_symbol_human_output(FILE *out, const ucl_object_t *obj)
+{
+	auto first = true;
+	double score = 0;
+	const char *key = nullptr, *desc = nullptr;
+
+	const auto *val = ucl_object_lookup(obj, "score");
+	if (val != nullptr) score = ucl_object_todouble(val);
+
+	key = ucl_object_key(obj);
+	val = ucl_object_lookup(obj, "description");
+	if (val != nullptr) desc = ucl_object_tostring(val);
+
+	std::string line = fmt::format("{:>4.1f} {:<22} ", score, key);
+	if (desc != nullptr) line += desc;
+
+	val = ucl_object_lookup(obj, "options");
+	if (val != nullptr && val->type == UCL_ARRAY) {
```

Comment:
```suggestion
	if (val != nullptr && ucl_object_type(val) == UCL_ARRAY) {
```

Reply:
This line was copied from rspamc_symbol_output() - so I have made above changes to this line as well as the line which was copied.